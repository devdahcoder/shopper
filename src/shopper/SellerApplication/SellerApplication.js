import React from 'react'
import "./assets/style/index.css"
import {BrowserRouter as Router, Switch, Route} from "react-router-dom"
import SellerApplicationNav from "../SellerApplicationNav/SellerApplicationNav"
import ApplicationCriteria from "../ApplicationCritaria/ApplicationCriteria"
import CompanyProfile from "../CompanyProfile/CompanyProfile"
import WebSocialMedia from "../WebSocialMedia/WebSocialMedia"
import ContactInfo from "../ContactInfo/ContactInfo"
import ConfirmAccount from "../ConfirmAccount/ConfirmAccount"

const SellerApplication = () => {
    return (
      <Router>
        <section>
          <div className="seller-application-display">
            <div style={{ textAlign: "center", padding: "10px 0" }}>
              <h1>Seller Application</h1>
            </div>

            <SellerApplicationNav />

            <Switch>
              <Route
                path="/application-criteria"
                exact
                component={ApplicationCriteria}
              />
              <Route 
                path="/company-profile" 
                exact 
                component={CompanyProfile} 
                
              />
              <Route
                path="/web-&-social-media"
                exact
                component={WebSocialMedia}
              />
              <Route 
                path="/contact-info" 
                exact 
                component={ContactInfo} 

              />
              <Route 
                path="/confirm-account" 
                exact 
                component={ConfirmAccount} 

              />
            </Switch>
          </div>
        </section>
      </Router>
    );
}

export default SellerApplication
