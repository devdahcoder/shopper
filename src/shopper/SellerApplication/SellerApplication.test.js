import React from "react";
import { render, cleanup } from "@testing-library/react";
import SellerApplication from "./SellerApplication"


afterEach(cleanup)

it("should take a snapshot", () => {
    const {asFragment} = render(<SellerApplication />)
    expect(asFragment(<SellerApplication />)).toMatchSnapshot()
})