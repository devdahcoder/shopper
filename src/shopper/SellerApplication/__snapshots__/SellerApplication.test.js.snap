// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <section>
    <div
      class="seller-application-display"
    >
      <div
        style="text-align: center; padding: 10px 0px;"
      >
        <h1>
          Seller Application
        </h1>
      </div>
      <section>
        <div
          class="seller-application-header"
        >
          <nav
            class="seller-nav"
          >
            <ul
              class="seller-ul"
            >
              <a
                href="/application-criteria"
              >
                <li>
                  Application Criteria
                </li>
              </a>
              <a
                href="/company-profile"
              >
                <li>
                  Company Profile
                </li>
              </a>
              <a
                href="/web-&-social-media"
              >
                <li>
                  Web & Social Media
                </li>
              </a>
              <a
                href="/contact-info"
              >
                <li>
                  Contact Info
                </li>
              </a>
              <a
                href="/confirm-account"
              >
                <li>
                  Confirm Account
                </li>
              </a>
            </ul>
          </nav>
        </div>
      </section>
    </div>
  </section>
</DocumentFragment>
`;
